Cargo.toml

```diff
@@ -71,7 +71,7 @@ mime_guess = "2.0.3"
 mozangle = "0.5.1"
 msg = { path = "components/shared/msg" }
 net_traits = { path = "components/shared/net" }
-num_cpus = { workspace = true }
+num_cpus = { version = "1.1.0" }
```

Comment:
This can just be:

`num_cpus = "1.1.0"`

Reply:
I updated it.